app/src/main/res/values/strings.xml

```diff
@@ -267,10 +267,10 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
 
     <string name="no_gps_no_quests">"To scan for quests around you automatically, turn on location"</string>
```

Comment:
Should this also be changed to "download data"?

Reply:
Hmm, I guess it makes sense to have consistent naming for the action everywhere.